internal/server/v2/facet/golden/contained_in_facet/CA_County_all.json

```diff
@@ -5,25 +5,45 @@
         "": {
           "ordered_facets": [
             {
-              "facet_id": "1145703171"
+              "facet_id": "2176550201",
+              "obs_count": 58,
+              "earliest_date": "1970",
+              "latest_date": "2021"
             },
             {
-              "facet_id": "1541763368"
+              "facet_id": "1145703171",
+              "obs_count": 58,
```

Comment:
why there is no earliest_date?

Reply:
I'm actually not sure, but some of the facet information from the cache returns no earliest_date ..

Reply:
I see, it was removed in https://github.com/datacommonsorg/mixer/pull/1295/files#diff-f92f6edcac465aee9633f109b36078f74067ded3b7f3fe7210be800fc60727a5. Can you add it back here?

Reply:
Actually, just checked, it's because infrequent table version is from before the earliest_date change. Next data release will fix this, which maybe I can do one this week (just the data release)

Reply:
I see. After that, maybe update the logic that relies on earliest_date only.

Reply:
added a todo